src/store/traction/ont/getters.js

```diff
@@ -164,4 +164,19 @@ export default {
       return []
     }
   },
+  runs: (state) => {
+    try {
+      return Object.values(state.resources.runs).map((r) => {
+        let instrument = Object.values(state.resources.instruments).find(
```

Comment:
think you can just do state.resources.instruments[r.ont_instrument_id]

Reply:
@harrietc52

Reply:
I think that would assume the position in the list is the same as the id, which might not be the case